src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionImpl.java

```diff
@@ -67,29 +62,4 @@ public void onAttached(Run<?, ?> run) {
     public void onLoad(Run<?, ?> run) {
         this.parent = run;
     }
-
-    @Restricted(NoExternalUse.class)
-    @SuppressWarnings("unused")
-    public ExternalWorkspace getDynamic(String token) {
```

Comment:
Now that the workspace browsing is resolved from the `RootAction`, this method is no longer needed.
I will not perform a separate release with #37, so there shouldn't be any binary compatibility issues.